code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -77,6 +77,9 @@
 	if(alien && alien == IS_SKRELL)
 		M.adjustToxLoss(2 * removed)
 		return
+	else if(istype(M.species, /datum/species/xenochimera)) //vorestation addition
+		affect_ingest(M, alien, removed) //like hell are chimera of all things going to be poisoned by injecting them with meat
```

Comment:
This is the affect_blood, not the affect_ingest. Protein isn't set as an injectable form of nutriment, so the ..() on the line below this one causes them to take tox from it. Can remove it though, it's not a huge amount.